src/app/components/confirmStxTransactionComponent/index.tsx

```diff
@@ -32,20 +33,19 @@ const Container = styled.div`
   margin-top: 22px;
   margin-left: 16px;
   margin-right: 16px;
-  overflow-y: auto;
```

Comment:
what's the reason for removing the scrollbar CSS code from here? 
we actually need this because with the removed CSS the confirm STX screen UI is a bit messed up at the bottom


https://github.com/secretkeylabs/xverse-web-extension/assets/88320460/bce21b3a-fbe6-4d20-a137-10c00a5007a9

Reply:
Oh I see, you're right. My intent here was to fix the cropped control buttons for this incoming tx signing popup:
![Monosnap Xverse Wallet 2023-09-01 16-13-23](https://github.com/secretkeylabs/xverse-web-extension/assets/36603049/662c153f-83ed-4285-a233-e81b95e0077c)

Reply:
I reverted this code change